src/ameba/rule/performance/first_last_after_filter.cr

```diff
@@ -45,7 +45,7 @@ module Ameba::Rule::Performance
     def test(source, node : Crystal::Call)
       return unless node.name.in?(CALL_NAMES) && (obj = node.obj)
       return unless obj.is_a?(Crystal::Call) && obj.block
-      return if !node.block.nil? || node.args.any?
+      return unless node.block.nil? && node.args.empty?
```

Comment:
using unless with multiple conditions is hard to read. This is not enforced but would be nice to avoid this imho. 

```suggestion
      return if if !node.block.nil? || !node.args.empty?
```

Reply:
I agree except for `&&` chaining which reads IMO pretty natural - `unless /all of the following/`